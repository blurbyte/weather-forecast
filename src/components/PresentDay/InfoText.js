import styled from 'styled-components';

import BaseText from '../Text';

const InfoText = styled(BaseText)`
  margin: 0.2rem 2rem 0.2rem 0;
  text-transform: none;

  &:last-child {
    margin-right: 0;
  }
`;

export default InfoText;
