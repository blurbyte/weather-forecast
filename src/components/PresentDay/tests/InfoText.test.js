import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import InfoText from '../InfoText';

test('renders correctly', () => {
  const component = renderer.create(<InfoText />);
  expect(component.toJSON()).toMatchSnapshot();
});
