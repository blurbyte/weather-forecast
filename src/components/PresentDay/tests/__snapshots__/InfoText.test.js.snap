// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c1 {
  font-size: 1.4rem;
  color: #222222;
  margin: 0;
  text-transform: none;
}

.c0 {
  margin: 0.2rem 1rem 0.2rem 1rem;
}

<p
  className="c0 c1"
/>
`;
